src/Membership/MembershipRequest.php

```diff
@@ -165,7 +161,7 @@ public function setEmailAddress(string $emailAddress): void
 
     public function getEmailAddress(): string
     {
-        return $this->emailAddress;
+        return $this->emailAddress ?: '';
```

Comment:
`?? '';`

Reply:
Why, in this case it's exactly the same thing no ?

Reply:
yes, in this case it do the same job, just for using php7